.github/workflows/integration.yml

```diff
@@ -1,16 +1,6 @@
 name: Integration tests
 
-on:
-  pull_request:
-    paths-ignore:
-      - 'README.md'
```

Comment:
Ignoring README.md feels like a logical move, but I just wondering why you decided to drop it?

Reply:
Just wanted to keep it simple and consistent with other `.yml` files. The savings (to Github) would be only a small amount, as the PR needs to be exclusively updating README.md to have the CI job not run.

Reply:
Got it, thanks!